CMakeLists.txt

```diff
@@ -200,7 +200,7 @@ endif()
 include(ProcessorCount)
```

Comment:
Do we have to comment this out? Or can we wrap it in an if-gate for fuzzing?

Reply:
Forgot to uncomment! It just sped up the build.